FlyingFox/Sources/WebSocket/SHA1.swift

```diff
@@ -96,6 +96,10 @@ extension SHA1 {
     }
 
     static func hash<D: DataProtocol>(data: D) -> Data where D.Index == Int {
+        // Convert the data to regular data in case it was a slice because slices can crash this hash algorithm
+        var data = Data()
+        data.append(data)
```

Comment:
otherwise we hash an empty data and get the null input `2jmj7l5rSw0yVb/vlWAYkK/YBwk=`

```suggestion
        let data = Data(data)
```